pkg/clusters/addons/kong/kong.go

```diff
@@ -146,6 +153,15 @@ func (a *Addon) Deploy(ctx context.Context, cluster clusters.Cluster) error {
 		)
 	}
 
+	imageRepo := fmt.Sprintf("image.repository=%s", DefaultEnterpriseImageRepo)
+	imageTag := fmt.Sprintf("image.tag=%s", DefaultEnterpriseImageTag)
+	if a.enterprise {
+		a.deployArgs = append(a.deployArgs,
+			"--set", imageRepo,
+			"--set", imageTag,
+		)
+	}
```

Comment:
This doesn't use the Builder repo/tag settings, correct? It doesn't look like the PR is using those at all yet--they're available, and `WithImage()` will set them, but the chart install will never make use of them.

I think what you want is to place the default constants in `builder.go`, then have `WithEnterprise()` set `b.repo` and `b.tag` to those defaults, and finally append `--set image.repository=<b.repo>` and `--set image.tag=<b.tag>` to install args. That sets the defaults while still allowing the user to override them by manually calling `WithImage()` if they prefer.

Reply:
Need configure environment variable under this repo persisting the license string for the security good; the kong-super-admin-secret also requires "password", should be fine since we do not have sensitive data for testing purpose. 

👍  your understanding about default and configure over default.

Reply:
OK. We do have such environment configuration 
https://github.com/Kong/kubernetes-testing-framework/settings/environments

Reply:
e02d137c22f069e1af34af42d9e3367a1e7e20b3